doc/builders/packages/weechat.section.md

```diff
@@ -0,0 +1,85 @@
+# Weechat
+
+Weechat can be configured to include your choice of plugins, reducing its closure size from the default configuration which includes all available plugins. To make use of this functionality, install an expression that overrides its configuration such as
+
+```nix
+weechat.override {configure = {availablePlugins, ...}: {
+    plugins = with availablePlugins; [ python perl ];
+  }
+}
+```
+
+If the `configure` function returns an attrset without the `plugins` attribute, `availablePlugins` will be used automatically.
+
+The plugins currently available are `python`, `perl`, `ruby`, `guile`, `tcl` and `lua`.
+
+The python and perl plugins allows the addition of extra libraries. For instance, the `inotify.py` script in weechat-scripts requires D-Bus or libnotify, and the `fish.py` script requires pycrypto. To use these scripts, use the plugin's `withPackages` attribute:
+
+```nix
```

Comment:
Looks like _pandoc_ choses to use indentation for codeblocks and doesn't have then the _nix_ language fence block.
I updated the PR and fixed up all the code blocks.